src/test/java/Entities/PropertyTest.java

```diff
@@ -146,7 +145,7 @@ public void testSetMortgageStatus() {
     public void testGetHouses() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Railroad",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,4,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,4,true);
         assertEquals(4, property1.getHouses());
     }
 
```

Comment:
No big comment here besides the fact that the tests might be a little excessive since it is testing all the getters and setters, but since they are already written, there is no big deal.